app/admin/reports/interval_data.rb

```diff
@@ -38,8 +38,31 @@
   filter :id
   filter :timestamp
 
+  contractor_filter :customer_id,
+                    label: 'Customer',
+                    path_params: { q: { customer_eq: true, ordered_by: :name } },
+                    if: proc { parent.group_by.include?('customer_id') }
```

Comment:
add `contractor_filter :vendor_id, ...` because vendor_id is present in `Report::IntervalCdr::CDR_COLUMNS`